setup/script-list.txt

```diff
@@ -0,0 +1,4 @@
+# List of setup scripts to execute with descriptions. Scripts are executed in order.
```

Comment:
We don't execute these with descriptions, could write just # Available scripts. You already mention how to add new scripts with descriptions below. Perhaps also a note under that to not forget to update run.sh.

Reply:
No need to update the script. Updated the rest.